daiquiri/core/generators.py

```diff
@@ -93,7 +93,7 @@ def generate_votable(generator, fields, infos=[], links=[], services=[], table=N
                                   .replace('>', '&gt;')
                 attrs.append('%s="%s"' % (key, value))
 
-        if 'ucd' in field:
+        if field.get('ucd'):
```

Comment:
I think this does not solve the issue, we need:
```python
if field.get('ucd', []):
```

can you confirm?

Reply:
The original fix solved the issue. I tested it on our dev instance. 

In my opinion, the `get` method doesn't need the default value `[]` , since we don't use  the returned value here anyway. Furthermore, the original bug is triggered if `field['ucd'] = None`. In this case, `field.get('ucd', []) = None` and the default value is not used.

Basically, my fix checks two things - (1) 'ucd' exists and (2) it's not `None`. I think that  it's sufficient at this point.

Reply:
my bad you are right.  I did not get that we were testing the return value of the get method. all fine.